src/App.vue

```diff
@@ -672,6 +672,41 @@ export default {
 				} else {
 					this.$store.dispatch('init', data)
 
+					if (data.deprecationWarning) {
+						await this.confirm(
+							'Z-Wave JS UI',
+							`<p>
+						After many misunderstandings and confusion, we have decided to rename <b>ZwaveJS2MQTT</b> to <b>Z-Wave JS UI</b>.
+						</p>
+						<p>
+						Many users were confused about 'MQTT' part thinking it needs an MQTT Broker to work when it's most important feature is to be a full featured Control Panel UI for Z-Wave JS.
+						We hope that this will make things more clear for everyone.
+						</p>
+						<p><b>
+						If you are seeing this message it means that you are using the old <code>zwavejs2mqtt</code> docker tag, please update to the new <code>zwave-js-ui</code> tag!.
+						</b></p>
+						<p>
+						There have been some breaking changes but nothing that should affect normal users:
```

Comment:
Maybe, since this doesn't affect normal users, just link to the release notes for more information at this point?

It prevents the user being overwhelmed with lots of info they probably don't need. The main focus of this dialog is informing of the name & tag change.

Reply:
Agree 👍🏼